src/config.ts

```diff
@@ -28,7 +28,7 @@ export default {
   database: {
     selected: 'mongo',
     mongoConfig: {
-      location: env.MONGO_HOSTNAME,
+      location: env.MONGO_URL,
```

Comment:
If we are going to change the name of the variable here, we will need to make sure to update the docker-compose files with the new variable name as well.